torchvision/datasets/imagenet.py

```diff
@@ -21,6 +21,10 @@
 class ImageNet(ImageFolder):
     """`ImageNet <http://image-net.org/>`_ 2012 Classification Dataset.
 
+    Before using this class, it is required to download `ImageNet 2012` dataset from
+    `here<https://image-net.org/challenges/LSVRC/2010/2010-downloads.php>`_ and
```

Comment:
Thanks for the PR @puhuk . Did you mean 2012 instead of 2010?

```suggestion
    `here<https://image-net.org/challenges/LSVRC/2012/2012-downloads.php>`_ and
```

Reply:
@NicolasHug Right, I resend the PR correcting this.